client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-table/index.tsx

```diff
@@ -12,22 +16,69 @@ interface Props {
 }
 
 export default function SiteTable( { isLoading, columns, items }: Props ) {
+	//const initialText = 'Edit All';
+	const translate = useTranslate();
+	const [ isBulkManagementActive, setIsBulkManagementActive ] = useState( true );
+
+	/**
+	 * this logic take the existing state and inverts it,
+	 * I've used the letter b because it's a Boolean
+	 */
+	const toggleBulkEditMode = () => setIsBulkManagementActive( ( b ) => ! b );
+
+	// Actions
+
 	return (
 		<table className="site-table__table">
 			<thead>
-				<tr>
-					{ columns.map( ( column, index ) => (
-						<th key={ column.key }>
-							{ index === 0 && (
-								<Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />
-							) }
-							<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
-								{ column.title }
-							</span>
+				{ isBulkManagementActive ? (
+					<tr>
+						{ columns.map( ( column, index ) => (
+							<th key={ column.key }>
+								{ index === 0 && (
+									<Icon className="site-table__favorite-icon" size={ 24 } icon={ starFilled } />
+								) }
+
+								<span className={ classNames( index === 0 && 'site-table-site-title' ) }>
+									{ column.title }
+								</span>
+							</th>
+						) ) }
+						<th>
+							<div className="plugin-common-table__bulk-actions">
+								<ButtonGroup className="edit-all__bulk-actions">
+									<Button compact onClick={ toggleBulkEditMode }>
+										{ translate( 'Edit All', { context: 'button label' } ) }
+									</Button>
+								</ButtonGroup>
+							</div>
 						</th>
-					) ) }
-					<th></th>
-				</tr>
+					</tr>
+				) : (
+					<tr>
+						<ButtonGroup className="edit-all__bulk-actions-pause-monitor">
+							<Button compact onClick={ toggleBulkEditMode }>
+								{ translate( 'Pause Monitor', { context: 'button label' } ) }
+							</Button>
+						</ButtonGroup>
+						<ButtonGroup className="edit-all__bulk-actions-resume-monitor">
```

Comment:
I don't think we need class name for `ButtonGroup`s

Reply:
I had to leave the classes, so I could style them appropriately.

Reply:
You are correct. I removed them as they were unnecessary.